### PR TITLE
Added v3 endpoint for was/folders

### PR DESCRIPTION
**Pull request by parvdesai-tenable**
# Description

Added v3 endpoint for was/folders

### CODE REVIEW

File: tenable/io/v3/was/folders.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+'''
+Folders
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`folders <folders>` API.
```
Comment (dhvanilshah-tenable) [inline tenable/io/v3/was/folders.py:6]: @harshshah-tenable @parvdesai-tenable We have folders endpoint in VM as well as WAS. Would we require to do that bifurcation here:
:devportal:`folders <folders>` API?

File: tenable/io/v3/was/folders.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        Args:
+            id: The unique identifier for the folder.
+
+        Returns:
+            :obj:`None`
+
+        Examples:
+            >>> tio.v3.was.folders.delete('91843ecb-ecb8-48a3-b623-d4682c28c')
+        '''
+        self._delete('{}/webapp'.format(id))
```
Comment (dhvanilshah-tenable) [inline tenable/io/v3/was/folders.py:58]: Please use f'' instead.

File: tenable/io/v3/was/folders.py (hunk 1 above)
Comment (parvdesai-tenable) [reply tenable/io/v3/was/folders.py]: Updated the documentation to point to WAS folders api

File: tenable/io/v3/was/folders.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+class FoldersAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/was/folders'
+    _conv_json = True
+
+    def create(self, name: str) -> Dict:
+        '''
+        Create a folder.
+
+        :devportal:`folders: create <folders-create>`
```
Comment (dhvanilshah-tenable) [inline tenable/io/v3/was/folders.py:28]: This should be was-v2-folders-create as per https://developer.tenable.com/reference#was-v2-folders-create

File: tests/io/v3/was/test_folders.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        json=SAMPLE_FOLDER
+    )
+    folder = api.v3.was.folders.create(SAMPLE_FOLDER['name'])
+    assert isinstance(folder, dict)
+    assert folder['folder_id'] == SAMPLE_FOLDER_ID
+
+
+@responses.activate
+def test_delete(api):
+    responses.add(
```
Comment (dhvanilshah-tenable) [inline tests/io/v3/was/test_folders.py:25]: Please add the doc strings in all tests.

Comment (parvdesai-tenable) [reply tests/io/v3/was/test_folders.py:31]: added doc strings

File: tenable/io/v3/was/folders.py (hunk 3 above)
Comment (parvdesai-tenable) [reply tenable/io/v3/was/folders.py]: updated

File: tenable/io/v3/was/folders.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    :members:
+'''
+import uuid
+from typing import Dict, List
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class FoldersAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/was/folders'
```
Comment (tomeara) [inline tenable/io/v3/was/folders.py:21]: There is still discussion on what the right naming convention should be here. It does seem that we will be moving forward with this structure though.

**Review by tomeara — APPROVED** [review]

**Review by dhvanilshah-tenable — APPROVED** [review]

**Pull request merged**